src/inmanta/module.py

```diff
@@ -2385,6 +2385,16 @@ def get_plugin_dir(self) -> Optional[str]:
         """
         raise NotImplementedError()
 
+    def _list_python_files(self, plugin_dir: str) -> Generator[None, str, None]:
```

Comment:
```suggestion
    def _list_python_files(self, plugin_dir: str) -> Iterator[str]:
```
No strong requirement for me but I find the `Iterator` type definition a lot more readable than the `Generator` one. And since this is actually an iterator we could just as well be specific.

Reply:
ah, is that correct? did not know that